src/components/FormDatePicker.vue

```diff
@@ -3,7 +3,8 @@
     <label v-uni-for="name">{{label}}</label>
     <date-picker
       :config="config"
-      v-model="date"
+      :value="moment(date)"
```

Comment:
This appears to be a bug with the date picker. Passing in a moment object allows the date picker to correctly display the date in its input.